images/miq-app/docker-assets/container-deploy-common.sh

```diff
@@ -67,6 +70,22 @@ fi
 
 }
 
+function check_version_gt() { 
+# Check if upgrade version is actually greater than stored PV version
+# -V sorts alphanumeric versions within text, will always return oldest version first
+# Compare sort version result against upgrade version
+
```

Comment:
a preliminary check can/should  be added here in  order to make sure we compare upstream with upstream and downstrem with downstream.
it could be done by checking the delimiter ('.' vs '-' or '').
although far fetched it will revile the extreme case  that is very hard to debug.